app/build.gradle

```diff
@@ -25,25 +24,24 @@ android {
 
         applicationId APPLICATION_ID
         versionCode 1
-        versionName '1.0-beta'
+        versionName '0.9.1'
         testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
         com.android.ddmlib.DdmPreferences.setTimeOut(60000)
+        javaCompileOptions.annotationProcessorOptions.includeCompileClasspath = true
```

Comment:
Is this parameter really necessary there? 
> If you are experiencing issues migrating to the new dependency resolution strategy, you can restore behavior to that of Android plugin 2.3.0 by setting includeCompileClasspath true. However, restoring behavior to version 2.3.0 is not recommended, and the option to do so will be removed in a future update.

Reply:
Yes. let's update butterknife in different PR